aiotg/bot.py

```diff
@@ -518,7 +519,13 @@ def __del__(self):
         await response.release()
 
     def _process_message(self, message):
-        chat = Chat.from_message(self, message)
+        chat_id = message["chat"]["id"]
+        chat = self._conversations.get(chat_id)
+
+        if not chat:
+            chat = Chat.from_message(self, message)
+        elif chat.is_conversation:
+            chat.update(message)
```

Comment:
I think better to leave all the logic in chat, what do you think?

Reply:
Honestly, it looked cleaner outside Chat. Now the whole feature feels invasive. Also, update() is a confusing name.